docs/source/using/experimental-features.adoc

```diff
@@ -73,6 +73,38 @@ If you prefer to use dynamic assignment, you can use:
 $ minishift ip --set-dhcp
 ----
 
+[[local-proxy-server]]
+== Local Proxy Server
+
+{project} provides an option to run a local proxy server on the host, which the {project} instance can use to access the internet.
+This can help in the situation when local security certificates are used in the organization, but can not easily be shared with the instance.
+
+Enabling the proxy server is done using the following command:
+
+----
+$ minishift config set local-proxy true
+----
+
+This will start a proxy server on the host which on start will automatically be assigned to the {project} instance.
+
+When the corporate or upstream proxy is known, you can specify this with the following configuration option:
+
+----
+$ minishift config set upstream-proxy [username:password@host:port]
+----
+
+When this option is used, all traffic will be re-encrypted with a {project} specific certificate. It is therefore advised not to use this proxy for any other purposes then development and use with {project}.
```

Comment:
"{project} specific" should be hyphenated: "{project}-specific".

In the second sentence, "then" should be "than".

I also feel like we can be more direct with the second sentence here, something like the following:

```adoc
For this reason, this proxy should only be used for development and use with {project}.
```

Also, please keep each sentence on its own line. :)